src/parser.js

```diff
@@ -380,10 +384,30 @@ export default class Parser {
         this.handleSwaggerMethod(resource, href, pathObjectParameters, methodValue, method);
       });
 
+      this.handleSwaggerVendorExtensions(resource, pathValue);
+
       return resource;
     });
   }
 
+  // Converts all unknown Swagger vendor extensions from an object into a API Element extension
+  handleSwaggerVendorExtensions(element, object) {
+    const extensions = _.chain(object)
+      .pick(isExtension)
+      .omit('x-description', 'x-summary', 'x-group-name')
```

Comment:
Since it's a vendor extension we've created and it's already translated to resource groups.